travis-yml/buffer-more-ints.travis.yml

```diff
@@ -0,0 +1,19 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : buffer-more-ints
+# Source Repo         : https://github.com/dpw/node-buffer-more-ints
+# Travis Job Link     : https://travis-ci.com/github/dthadi3/node-buffer-more-ints/builds/211017296
+# Created travis.yml  : No
+# Maintainer          : Devendranath Thadi <devendranath.thadi3@gmail.com>
+#
+# Script License      : MIT License
+#
+# ----------------------------------------------------------------------------
+
+arch:
+  - amd64
+  - ppc64le
+language: node_js
+node_js:
+  - "8"
```

Comment:
The node versions seem quite old and out of date, did we try updating these to the latest?

Reply:
Yeah, I tried to update latest node_js versions 14 & 15, both amd64 & ppc64le jobs are failing with DeprecationWarning.
https://travis-ci.com/github/dthadi3/node-buffer-more-ints/builds/211269183